test/integration_test.sh

```diff
@@ -35,34 +34,42 @@
 # lira_mode and lira_version
 # The lira_mode param can be "local", "image" or "github".
 # If "local" is specified, a local copy of the Lira code is used and
-# lira_version is ignored. If "image" is specified, this script will pull and run
+# lira_version is ignored.
+# 
+# If "image" is specified, this script will pull and run
 # a particular version of the Lira docker image specified by lira_version.
-# TODO: Modify so that if lira_mode == "image" and lira_version == "deployed",
-# then the script will use the currently deployed image in env.
-# Running in "github" mode is not fully implemented yet, but is intended to clone
-# the Lira repo and check out a specific branch, tag, or commit to use, specified
-# by lira_version.
+# If lira_version == "latest_released", then the script will scan the GitHub repo
+# for the highest tagged version and try to pull an image with the same version.
+# If lira_version == "latest_deployed", then the script will use the latest
+# deployed version in env, specified in the deployment tsv. If lira_version is
+# any other value, then it is assumed to be a docker image tag version and
+# this script will attempt to pull that version.
+#
+# Running in "github" mode causes this script to clone the Lira repo and check
+# out a specific branch, tag, or commit to use, specified by lira_version.
 #
 # pipeline_tools_mode and pipeline_tools_version
```

Comment:
Just want to make sure, pipeline_tools_mode will only have `local` and `github` now right?

Reply:
Yes, that's correct

Reply:
Thanks!